providers/gcp.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2025
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: gcp
+clusterGVKs:
+  - group: infrastructure.cluster.x-k8s.io
+    version: v1beta2
```

Comment:
is that correct? i can only see the v1beta1 version of the GCPCluster. the same goes with the GCPManagedCluster kind

Reply:
Right. My mistake. I'll fix that. Thanks, Michael.